File: headhunter/Entities/Vacancy.cs
namespace headhunter.Entities
{
    public class Vacancy : BaseEntity
    {
        public string Position { get; set; }
        public string Description { get; set; }
        public int Salary { get; set; }
        public string Department { get; set; }
        public string Location { get; set; }
        public string Requirements { get; set; }
        public string Responsibilities { get; set; }
        public string Email { get; set; }
        public string CompanyName { get; set; }
        public int CompanyId { get; set; }
    }
}

File: client/client/src/app/interfaces/vacancy.ts
export interface ICompany {
  id: number;
  name: string;
  industry: string;
  foundedYear: string;
  description: string;
  vacancies: IVacancy[];
}

export interface IVacancy {
  id: number;
  position: string;
  salary: number;
  department: string;
  location: string;
  requirements: string;
  responsibilities: string;
  email: string;
  description: string;
  companyName: string;
  companyId: number;
}
